tests/subsys/logging/log_stress/testcase.yaml

```diff
@@ -0,0 +1,62 @@
+common:
+  filter: CONFIG_QEMU_TARGET and not CONFIG_SMP
+  tags: log_api logging
+  integration_platforms:
+    - qemu_x86
+tests:
+  logging.log_stress_light_v1:
+    extra_configs:
+      - CONFIG_LOG1=y
+      - CONFIG_LOG_MODE_DEFERRED=y
```

Comment:
minor nit. from what I can tell `CONFIG_LOG_MODE_DEFERRED=y` is used on all tests.